radio/src/telemetry/frsky.cpp

```diff
@@ -622,6 +631,15 @@ void telemetryInit(void)
   else {
     telemetryPortInit(FRSKY_SPORT_BAUDRATE);
   }
+
+#if defined(REVX)
```

Comment:
Drats I can totally see why. 'learn how to build and use the simulator' is somewhere on my TODO list.